matrimony_admin/views.py

```
@@ -1,16 +1,22 @@
+from django.views.generic import TemplateView, DetailView, ListView,DeleteView,UpdateView
 from django.contrib import messages
 from django.contrib.auth import authenticate, login, logout
-from django.views.generic import FormView
+from django.views.generic import FormView, ListView
 from django.urls import reverse_lazy
-from .forms import AdminLoginForm, AdminProfileForm, NotificationDetailsForm
+from .forms import AdminLoginForm, AdminProfileForm,NotificationDetailsForm, EditUserForm
 from .models import BlockedUserInfo
 from U_auth.permissions import *
-
+from U_auth.models import UserPersonalDetails, Location
```

Ruff

`U_auth.models.Location` imported but unused

Remove unused import: `U_auth.models.Location`

(F401)